app/routes/index.js

```diff
@@ -33,7 +33,7 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin,  {
             if (this.get('theme.isProvider'))
                 route = `provider.${route}`;
 
-            this.transitionTo(route, { queryParams: { queryString: q } });
+            this.transitionTo(route, { queryParams: { q: q } });
```

Comment:
Super minor, does not actually have to be changed, but I find easier to read it like this:  `{queryParams: { q } }`